app-misc/ckb/ckb-9999.ebuild

```diff
@@ -0,0 +1,73 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit cmake udev xdg
+
+DESCRIPTION="Corsair K65/K70/K95 Driver"
+HOMEPAGE="https://github.com/ckb-next/ckb-next"
+
+if [[ ${PV} == "9999" ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/ckb-next/ckb-next.git"
+else
+	SRC_URI="https://github.com/ckb-next/ckb-next/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+	KEYWORDS="~amd64"
+	S="${WORKDIR}/${PN}-next-${PV}"
+fi
+
+LICENSE="GPL-2"
+SLOT="0"
+IUSE="systemd"
+
+RDEPEND="
+	dev-libs/libdbusmenu-qt
+	dev-libs/quazip:0=
+	dev-qt/qtcore:5
+	dev-qt/qtdbus:5
+	dev-qt/qtgui:5
+	dev-qt/qtnetwork:5
+	dev-qt/qtwidgets:5
+	dev-qt/qtx11extras:5
+	|| (
+		media-sound/pulseaudio
+		media-sound/apulse[sdk]
+	)
+	virtual/libudev:=
+	x11-libs/libX11
+	x11-libs/libxcb:=
+	x11-libs/xcb-util-wm"
+DEPEND="${RDEPEND}"
+BDEPEND="dev-qt/linguist-tools:5"
+
+src_configure() {
+	local mycmakeargs=(
+		-DDISABLE_UPDATER=yes
+		-DFORCE_INIT_SYSTEM=$(usex systemd systemd openrc)
+	)
+	cmake_src_configure
+}
+
+src_install() {
+	cmake_src_install
+	dodoc CHANGELOG.md
```

Comment:
Why remove the init file?

Reply:
The build system ships it own file from the upstream archive. Without removing this one, two different initd files would be installed at `/etc/init.d/ckb-daemon` and `/etc/init.d/ckb-next-daemon`. The former one from the portage tree, tries to start `/usr/bin/ckb-daemon` which wouldn't work since the daemon is installed as `/usr/bin/ckb-next-daemon`. Maybe the initrd file from the tree is outdated now?

Reply:
Ah, you're right. Interesting that no one's reported a bug for that.